src/GitHubExtension/DeveloperId/DeveloperIdProvider.cs

```diff
@@ -54,7 +54,15 @@ private DeveloperIdProvider()
             DeveloperIds ??= new List<DeveloperId>();
 
             // Retrieve and populate Logged in DeveloperIds from previous launch.
-            RestoreDeveloperIds(CredentialVault.GetAllSavedLoginIds());
+            try
+            {
+                RestoreDeveloperIds(CredentialVault.GetAllSavedLoginIds());
+            }
+            catch (Exception ex)
+            {
+                // Log error and continue even if Restore of previously logged in DevIds failed.

```

Comment:
Should there be any user experience explaining why IDs are missing from this launch? a notification or warning in the tool indicating there was an account issue?